### DISCUSSION TITLE
Click action does not redirect to the route passed

### DISCUSSION THREAD

**Discussion by dotmarn** [kutia-software-company/larafirebase#29] [Q&A] [answered]
This package is very lovely as it make push notification to work without so much hassle but whenever I click on the notification message received, it only focus on the webpage but not redirect to the route passed. Any idea?

**Comment by gentritabazi** [comment, marked as answer]
Based on the Firebase [documentation](https://firebase.google.com/docs/cloud-messaging/http-server-ref#notification-payload-support), For all URL values, HTTPS is required.

![image](https://user-images.githubusercontent.com/35135482/179604914-90d9c210-cba4-4d44-8bd6-89e0744355fe.png)
